lib/addons/kubevious/index.ts

```diff
@@ -0,0 +1,66 @@
+import { Construct } from '@aws-cdk/core';
+import { HelmAddOn, HelmAddOnUserProps, HelmAddOnProps } from "../helm-addon";
+import { ClusterInfo, Values } from "../../spi";
+import { setPath } from "../../utils";
+
+
+/**
+ * User provided options for the Helm Chart
+ */
+export interface KubeviousAddOnProps extends HelmAddOnUserProps {
+    version?: string,
+    ingressEnabled?: boolean,
+    kubeviousServiceType?: string,
+    mysqlRootPassword?: string,
+}
+
+/**
+ * Default props to be used when creating the Helm chart
+ */
+const defaultProps: HelmAddOnProps & KubeviousAddOnProps = {
+    name: "ssp-kubevious-addon",
+    namespace: "kubevious",
+    chart: "kubevious",
+    version: "0.8.15",
+    release: "kubevious",
+    repository:  "https://helm.kubevious.io",
+    values: {},
+
+    ingressEnabled: false,
+    kubeviousServiceType: "ClusterIP",
+    mysqlRootPassword: "kubevious"
```

Comment:
Seems like these are defaults. Let customers pass `values` if they wish to override.

Reply:
Hi  @askulkarni2 , the defaultProps contains the defaults while the user can pass override value props on the class constructor. The default values and the user provided values are loaded into this.options. The class uses this.options to instantiate the add-on . As such the user can override the defaults already. Please let me know if I missunderstood the request

Reply:
Yes, that is the intention. Default values mimic the values in the external props allowing customers to overwrite. I think we are good.